weldx/tags/time/timedeltaindex.py

```diff
@@ -8,6 +8,15 @@
 
 __all__ = ["TimedeltaIndexConverter"]
 
+PANDAS_OLD_UNIT_SUFFIXES = dict(H="h", T="min", S="s", L="ms", U="us", N="ns")
+
+
+def _handle_converted_pd_tdi_units(node: TaggedDict):
+    """Convert changed units in Pandas.Datetimeindex to valid values."""
+    unit = node["freq"][-1]
+    if unit in PANDAS_OLD_UNIT_SUFFIXES.keys():
+        node["freq"] = node["freq"][:-1] + PANDAS_OLD_UNIT_SUFFIXES[unit]
```

Comment:
```suggestion
    for suf in PANDAS_OLD_UNIT_SUFFIXES:
        node["freq"] = node["freq"].replace(suf, PANDAS_OLD_UNIT_SUFFIXES[suf])
```
I think this should be more stable @marscher ?

Reply:
I'm a bit confused about the stability here. Can you help me understand how a frequency string could have multiple time units? Is that even possible? Also, I assume it only covers cases where the frequency string is less than one character long, correct?

Reply:
Multiple frequency "units" can be combined to form a single timedelta:
```python
pd.timedelta_range(start=0, periods=5, freq="2D10L")
```
Not sure if there is a way to end up in the file like that but since the old strings are one-character only I think the replace option should cover everything.